Slim/Control/Commands.pm

```diff
@@ -3078,7 +3078,7 @@ sub syncCommand {
 		# try a player index
 		if (!defined $buddy) {
 			my @clients = Slim::Player::Client::clients();
-			if (defined $clients[$newbuddy]) {
+			if (looks_like_number($newbuddy) && defined $clients[$newbuddy]) {
 				$buddy = $clients[$newbuddy];
```

Comment:
Please move the new check to the outer condition. There's no need to read the `@clients` list if the `$newbuddy` isn't a number.

And feel free to update `Changelog8.html` too.

Reply:
I have updated the PR